code-with-comment.html

```diff
@@ -36,7 +44,21 @@
           type: Number,
           notify: true
         },
-        indexItem: Number
+        indexItem: Number,
+        hideAddComments: {
+          type: Boolean,
+          value: true
+        },
+        comments: Array,
+        hideComments: Boolean
+      },
+      _splitCode: function (e) {
+        this.hideAddComments = false;
+        e.stopPropagation();
+        var elem = e.detail;
+        elem.comments = this.comments;
+        this.fire('add-hunk-with-comment', elem);
+        this.comments = [];
       }
```

Comment:
Given my comment above, this element will become responsible for:
 - checking the click on the last line
 - editing the `code` property
 - firing the `add-code-with-comment` event to the parent